source/runner/Dangerfile.ts

```diff
@@ -62,10 +67,10 @@ export function contextForDanger(dsl: DangerDSLType): DangerContext {
     markdowns: []
   }
 
-  const fail = (message: MarkdownString) =>  results.fails.push({ message })
-  const warn = (message: MarkdownString) => results.warnings.push({ message })
-  const message = (message: MarkdownString) => results.messages.push({ message })
-  const markdown = (message: MarkdownString) => results.markdowns.push(message)
```

Comment:
`results.markdowns: Array` differs in that it currently accepts `MarkdownString` as its type, whereas `fails`, `warns`, and `messages` accept `Violation` as its type. To keep parity with the Danger-rb API, `markdown()` should be able to accept `{ file, line }` options, so I believe moving `markdowns` from `Array<MarkdownString>` to `Array<Violation>` is the appropriate change.